website/docs/cli/commands/test.mdx

```diff
@@ -43,3 +45,13 @@ The Terraform `test` command creates _real_ infrastructure. Once Terraform fully
 You should monitor the output of the test command closely to ensure Terraform removes the infrastructure it created or perform manual cleanup if not. We recommend creating dedicated testing accounts within the target providers that you can routinely and safely purge to ensure any accidental and costly resources aren't left behind.
 
 Terraform also provides diagnostics explaining why it could now automatically clean up. You should resolve these diagnostics to ensure that future clean-up operations are successful.
+
+## Terraform Cloud Execution
+
+You can execute your tests remotely on Terraform Cloud using the `-cloud-run` option.
+
+The argument accepts a private registry module source as specified in the [Terraform Registry Module Sources](/terraform/language/modules/sources#terraform-registry) documentation. This will associate the test run with the given private module within the Terraform Cloud UI.
```

Comment:
Can we shorten this to:
```suggestion
The `-cloud-run` command accepts a [private registry module source](/terraform/language/modules/sources#terraform-registry) as as argument. This argument associates the test run with your specified private module within the Terraform Cloud user interface.
```

Reply:
Tweaked this a bit more to get around confusino of command vs argument vs option:

```
The `-cloud-run` option accepts a [private registry module source](/terraform/language/modules/sources#terraform-registry). This option associates the test run with your specified private module within the Terraform Cloud user interface.
```